src/components/components/components/components/components/components/AddPostForm.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+
+const initialForm = {
+  title: "",
+  content: "", 
+  contactId: undefined
+}
+
+export default function AddPostForm({setReloadPostList, reloadPostList}) {
+  const [form, setForm] = useState(initialForm)
+
+
+
+  // simulating a logged in user
+  // const loggedInUserId = 1
+
+  const postNewPost = () => {
+    const options = {
+      method: "POST",
+      headers: {"content-type":"application/json"},
+      body: JSON.stringify(form)
+    }
+
+    fetch("https://boolean-api-server.fly.dev/Chloe070196/post", options)
+    .then(r => r.json())
+    .catch(error => console.log(error))
+    setForm(initialForm)
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if (!form.title) {
+      form.title = form.content.slice(0, 20) + "..."
+    }
+    postNewPost()
+    setReloadPostList(true)
```

Comment:
I found a bug, when making a new post on the feed. The comments seem to stay put, so basically the comment for the last made post now have attached themselves to the new one. I believe you should also add `setReloadComments(true)` here as well.

Reply:
I'll have a look, glad you spotted that (it was an issue, then it wasn't - looks like it's come back to haunt me)